lib/ontologies_linked_data/config/config.rb

```diff
@@ -103,6 +103,88 @@ def config(&block)
     # number of threads to use when indexing a single ontology for search
     @settings.indexing_num_threads          ||= 1
 
+
+
+    # Global connector configuration
+    @settings.connectors ||= {
+      available_sources: {
+        'ANR' => Connectors::AnrConnector,
+        'CORDIS' => Connectors::CordisConnector
+      }
+    }
+    @settings.project_sources ||= ["ANR", "CORDIS"]
+    @settings.project_types ||= ["FundedProject", "NonFundedProject"]
+
+    # CORDIS Connector Configuration
+    @settings.cordis_connector ||= {
```

Comment:
here it will be to have a list of connectors , instead of doining `@settings.cordis_connector` and `@settings.anr_connector` 

for example like this
```
@setttings.connectors = {
   cordis: {your mappings object},
  anr: {your mapping },
}
```